ci-operator/step-registry/hypershift/mce/kubevirt/create/disconnected/hypershift-mce-kubevirt-create-disconnected-chain.yaml

```diff
@@ -0,0 +1,207 @@
+        mkdir -p /home/release-manifests/
+        oc image extract \${PAYLOADIMAGE} --path /release-manifests/:/home/release-manifests/ --confirm
+        RHCOS_IMAGE=\$(cat /home/release-manifests/0000_50_installer_coreos-bootimages.yaml | yq -r .data.stream | jq -r '.architectures.x86_64.images.kubevirt."digest-ref"')
+        RHCOS_IMAGE_NO_DIGEST=\${RHCOS_IMAGE%@sha256*}
+        RHCOS_IMAGE_NAME=\${RHCOS_IMAGE_NO_DIGEST##*/}
+        RHCOS_IMAGE_REPO=\${RHCOS_IMAGE_NO_DIGEST%/*}
+
+        oc image mirror \${RHCOS_IMAGE} \${mirror_registry}/\${LOCALIMAGES}/\${RHCOS_IMAGE_NAME}
+
+        oc apply -f - <<EOF2
+        apiVersion: operator.openshift.io/v1alpha1
+        kind: ImageContentSourcePolicy
+        metadata:
+          name: openshift-release-dev
+        spec:
+          repositoryDigestMirrors:
+            - mirrors:
+                - \${mirror_registry}/\${LOCALIMAGES}
+              source: \${RHCOS_IMAGE_REPO}
+        EOF2
+
+        ####
+
+        arch=\$(arch)
+        if [ "\$arch" == "x86_64" ]; then
+        downURL=\$(oc get ConsoleCLIDownload hcp-cli-download -o json | jq -r '.spec.links[] | select(.text | test("Linux for x86_64")).href') && curl -k --output /tmp/hcp.tar.gz \${downURL}
+        cd /tmp && tar -xvf /tmp/hcp.tar.gz
+        chmod +x /tmp/hcp
+        cd -
+        fi
+
+        if [ ! -f /tmp/yq-v4 ]; then
+        curl -L "https://github.com/mikefarah/yq/releases/download/v4.30.5/yq_linux_\$(uname -m | sed 's/aarch64/arm64/;s/x86_64/amd64/')" \
+        -o /tmp/yq-v4 && chmod +x /tmp/yq-v4
+        fi
+        oc get imagecontentsourcepolicy -oyaml | /tmp/yq-v4 '.items[] | .spec.repositoryDigestMirrors' > /home/mgmt_iscp.yaml
+        ICSP="--image-content-sources /home/mgmt_iscp.yaml "
+
+        HYPERSHIFT_NODE_COUNT=\$(cat /home/hypershift_node_count)
+
+        CLUSTER_NAME=\$(cat /home/hostedcluster_name)
+        CLUSTER_NAMESPACE=local-cluster-\${CLUSTER_NAME}
+        echo "\$(date) Creating HyperShift cluster \${CLUSTER_NAME}"
+        oc create ns "\${CLUSTER_NAMESPACE}"
+        BASEDOMAIN=\$(oc get dns/cluster -ojsonpath="{.spec.baseDomain}")
+        echo "extract secret/pull-secret"
+        oc extract secret/pull-secret -n openshift-config --to=/tmp --confirm
+
+        echo "extract secret/pull-secret"
+        oc extract secret/pull-secret -n openshift-config --to=/tmp --confirm
+        PULL_SECRET_PATH="/tmp/.dockerconfigjson"
+        TRUST="--additional-trust-bundle /etc/pki/ca-trust/source/anchors/registry.2.crt "
+
+        # TODO: can we avoid this?
+        ROVERRIDES=\$(oc get imagecontentsourcepolicy image-policy -o=json | jq -r '.spec.repositoryDigestMirrors[] | select (.source | test("registry.ci.openshift.org/ocp/") ) | .source+"="+.mirrors[0]')
+        oc patch deploy -n hypershift operator --type=json -p="[{\"op\": \"add\", \"path\": \"/spec/template/spec/containers/0/args/-\", \"value\": \"--registry-overrides=\${ROVERRIDES}\"}]"
+        sleep 30
+        ###
```

Comment:
This shouldn't be necessary. It should be enough to pass the `--image-content-sources` during `hcp create`.

Reply:
OK, let's try skipping this